.travis.yml

```diff
@@ -1,9 +1,24 @@
 ---
+sudo: required
 services: docker
+language: generic
+dist: xenial
 
 branches:
   only:
   - master
   - travis
 
-script: echo "Hello, World!"
+jobs:
+  include:
+    - stage: tests
+      script:
+      # Start containers
+      - docker-compose build
+      - docker-compose up -d
+      # Getting ready to test
+      - echo "127.0.0.1 dashboard.openwisp.org controller.openwisp.org radius.openwisp.org topology.openwisp.org" | sudo tee -a /etc/hosts
+      - sleep 1m
```

Comment:
why sleep for 1 minute? Is not clear. Maybe we need a comment, or maybe we don't need to wait 1 minute, or maybe there's a better way to understand when we can get started to test, @2stacks @hispanico do you have any suggestion on this front?

Reply:
Indeed. Sleep is there to ensure all the services are running.
Not just the postgresql but also the `migrations` and `collectstatic`.
Another way would be loop a cURL request until I get 200 on the login page. :-)

Reply:
the loop can work with a time out or max number of attempts, I like that better than the sleep